scripts/plotting/create_active_sampler_learning_plots.py

```diff
@@ -40,19 +40,19 @@ def _derive_per_task_average(metric: str,
                              unsolved_task_penalty: float = 100) -> float:
     """Add a large constant penalty for unsolved tasks."""
     total_tasks = int(row["num_test_tasks"])
-    total_options_executed = 0.0
-    for test_task_idx in range(1, total_tasks + 1):
+    total_value = 0.0
+    for test_task_idx in range(total_tasks):
```

Comment:
this range change was the bug, the other changes are just fixing bad variable names